invokeai/app/invocations/latent.py

```diff
@@ -385,8 +423,16 @@ class LatentsToLatentsInvocation(TextToLatentsInvocation):
     type: Literal["l2l"] = "l2l"
 
     # Inputs
+    noise: Optional[LatentsField] = Field(description="The noise to use (test override for future optional)")
+
+    denoising_start: float = Field(default=0.0, ge=0, le=1, description="")
+    #denoising_end: float = Field(default=1.0, ge=0, le=1, description="")
+
     latents: Optional[LatentsField] = Field(description="The latents to use as a base image")
-    strength: float = Field(default=0.7, ge=0, le=1, description="The strength of the latents to use")
+
+    mask: Optional[ImageField] = Field(
+        None, description="Mask",
+    )
```

Comment:
So when we think of `inpaint`, we are thinking of a "workflow". And our `l2l` logic optional accepts a mask.

What do you get when you `l2l + mask`, but without the rest of the `inpaint` workflow? Just a crappy inpaint?

Reply:
Yes, but in reality - it's not that crappy as denoising take in account, original surrounding pixels.
For example you can look at this picture. It's l2l + mask on normal image with anime model, so completely incompatible, but you can't see sharp border.
![2361ffe1-d63f-4808-b181-6c649e0ed03f](https://github.com/invoke-ai/InvokeAI/assets/7768370/64776af7-0188-4e18-86f8-494d3d52b677)